src/main/java/dev/jensderuiter/minecraft_imagery/image/ImageCapture.java

```diff
@@ -338,7 +339,7 @@ private RayTraceResult rayTraceEntitiesFromList(Location start, Vector direction
             }
         }
 
-        return nearestHitEntity == null ? null : new RayTraceResult(nearestHitResult.getHitPosition(), nearestHitEntity, nearestHitResult.getHitBlockFace());
+        return (nearestHitEntity == null || nearestHitResult == null) ? null : new RayTraceResult(nearestHitResult.getHitPosition(), nearestHitEntity, nearestHitResult.getHitBlockFace());
```

Comment:
Why would this be needed?

Reply:
Suppresses linter warnings, and has no real impact on the code